backend/siarnaq/api/teams/views.py

```diff
@@ -173,3 +176,28 @@ def avatar(self, request, pk=None, *, episode_id):
             titan.upload_image(avatar, profile.get_avatar_path())
 
         return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+    @extend_schema(responses={status.HTTP_204_NO_CONTENT: None})
+    @action(
+        detail=False,
+        methods=["post"],
+        serializer_class=TeamReportSerializer,
+        permission_classes=(IsAuthenticated, IsEpisodeAvailable),
+    )
+    def report(self, request, pk=None, *, episode_id):
```

Comment:
Not super important, but I might suggest copying over some of the structure of `UserViewSet.resume`, which would let them download their report too! Would essentially consist of changing `methods` to `["get", "put"]` and calling `titan.get_object` in the `get` case.